django/db/backends/base/features.py

```diff
@@ -297,7 +297,8 @@ class BaseDatabaseFeatures:
     # field(s)?
     allows_multiple_constraints_on_same_fields = True
 
-    # Does the backend support boolean expressions in the SELECT clause?
+    # Does the backend support boolean expressions in SELECT and GROUP BY
+    # clauses?
```

Comment:
I don't think it's worth to add a second flag for the `GROUP BY` clause. IMO we can reuse `supports_boolean_expr_in_select_clause`.